modules/integration_gcp-firebase/conf/01-database_load.yaml

```
@@ -0,0 +1,18 @@
+module: "GCP Firebase database"
+name: "Firebase database load"
+
+transformation: ".min(over='30m')"
```

It's best to use lasting instead of min() / max() / ... It makes graphs easier to read with real time value
(if appropriate)

```suggestion
transformation: true
```